pkg/cmd/flagtypes/addr.go

```diff
@@ -106,6 +106,8 @@ func (a *Addr) Set(value string) error {
 
 	if !a.AllowPrefix {
 		addr.Path = ""
+	} else if addr.Path == "/" {
```

Comment:
The behavior of url Parse is to always set this I believe.  I'm not sure we should be clearing it here.